### ISSUE TITLE
Avoid 'checkout error' mail for 'ci skip' changes

### ISSUE THREAD

**Issue by mauritsvanrees** [plone/jenkins.plone.org#128]
When you push a change to a package that is not in checkouts.cfg you get a mail like this:

  Subject: CHECKOUT ERROR plone/Products.MimetypesRegistry master

```
Repository: plone/Products.MimetypesRegistry
Author: mauritsvanrees &lt;maurits@vanrees.org&gt;
Plone Version: 5.0

This package is used by Plone coredev 5.0 branch, 
but it’s not currently checked out so is not including
your change for testing.

You must add this package at 

https://github.com/plone/buildout.coredev/blob/5.0/checkouts.cfg”
```

Can we switch this off for commits that have `[ci skip]` in the commit message? I did two bulk edits this week, with a small change in almost all repos. That leads to a flood of emails.

I don't know where the script lives that does this. I cannot find it in this repository.

BTW, note the quote mark at the end of the mail. This should be removed, so the link works.

**Comment by gforcada** [comment]
That's on [mr.roboto](https://github.com/plone/mr.roboto/), and yes it needs quite some love :-)

**Comment by gforcada** [comment]
Jenkins can do that too: https://wiki.jenkins-ci.org/display/JENKINS/Ci+Skip+Plugin

**Comment by gforcada** [comment]
Sorry the previous was unrelated, it needs a fix on mr.roboto...

**Comment by mauritsvanrees** [comment]
I see the superfluous quote at the end of the mail, that I mentioned above, has been fixed by @gforcada in 6ff0012d7044371d1c711aa4b84ad22bc5627438. Thanks.
See the mr.roboto pull request from two minutes ago for the rest.
